projectile-rails.el

```diff
@@ -518,22 +532,29 @@ The bound variable is \"filename\"."
                                           "/[0-9]\\{14\\}.*_\\(${plural}\\|${singular}\\).*\\.rb$"
                                           'projectile-rails-find-migration))
 
+(defcustom projectile-rails-resource-name-re-list
+  `("/app/models/\\(?:.+/\\)?\\(.+\\)\\.rb\\'"
```

Comment:
It is the paranoid version of `$`.  `\'` only matches at the end of a whole string whie `$` also matches before a newline character, if any.